integration-tests/sceptre-project/config/external-stack-output/resolver-with-profile-region.yaml

```diff
@@ -0,0 +1,6 @@
+template:
+  path: dependencies/dependent_template_local_export.json
+
+
+parameters:
+  DependentStackName: !stack_output_external "{{project_code}}-external-stack-output-outputter::StackName {{environment_variable.AWS_PROFILE|default('default')}}::eu-west-1"
```

Comment:
This is what used to be 9/B.

The changes here fix issues running the integration tests locally:
1. We use jinja syntax to reference the project code instead of what we used to do, which was modify this template before tests were executed. Which was weird and unnecessary and produced a diff every time you ran these tests locally.
2. Before, it assumed you had a profile called "default" set up. But running this locally, that might not actually be the profile you're using. So now, it uses your AWS_PROFILE environment variable, if it's set; otherwise, it'll default to "default".